docs/linking/3-authentication-otp.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+
+title PISP Linking: Authentication (OTP)
+
+participant "PISP" as PISP
+
+box "Mojaloop"
+    participant Switch
+end box
+
+participant "DFSP" as DFSP
+
+autonumber 1 "<b>AUTH-#</b>"
+
+...
+
+note over PISP, DFSP
+  Here the user provides the OTP sent directly to them by the DFSP into the PISP App. It's then used as the secret to prove to the DFSP that the user trusts the PISP.
+end note
+
+...
+
+PISP -> Switch ++: ""PUT /consentRequests/456 {""\n\
+    ""  initiatorId: "PISP",""\n\
+    ""  authChannels: [ "OTP" ],""\n\
+    ""  scopes: [ ""\n\
+    ""    { accountId: "dfsp.username.1234",""\n\
+    ""      actions: "accounts.transfer", "accounts.getBalance" },""\n\
```

Comment:
Is actions a JSON array?
```suggestion
    ""      actions: ["accounts.transfer", "accounts.getBalance"] },""\n\
```

Reply:
Yes -- Let me do a much much larger find-and-replace here because I'm guessing this mistake is *everywhere*.